src/control/cmd/daos/util.h

```diff
@@ -33,6 +33,7 @@
 #include <dfuse_ioctl.h>
 
 #include "daos_hdlr.h"
+#include "dfs_internal.h"
```

Comment:
Is this needed?  Perhaps the include could be added directory to the acl.go file directory?  I'm not that good at includes from go code however.

Reply:
+1 to not accessing a private internal header from here, is there anyway to move what is needed externally to a public header

Reply:
i do not want to expose that as a public function for now.

Reply:
> You should move this include to the top of acl.go, where util.h is already included. You can add it as another #include line there. However it seems odd to me to include a private header from another directory. If the daos tool is using the dfs function, it seems like it should be public.

just FYI, adding that at top of acl.go, the build error is:
scons: *** [build/debug/gcc/src/control/daos] Implicit dependency `src/control/cmd/daos/dfs_internal.h' not found, needed by target `build/debug/gcc/src/control/daos'.